src/utils/useUpdateEffect/useUpdateEffect.ts

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+// useUpdateEffect is a useEffect that doesn't fire on initial render
+// for detecting when values change
+// https://stackoverflow.com/a/58217148
+
+const useIsMounted = function useIsMounted() {
+    const isMounted = React.useRef(false);
+
+    React.useEffect(function setIsMounted() {
```

Comment:
eslint is not liking the named function instead of an anonymous one. I normally would name a function if it was resolving an event and I wanted to remove the event listener, but what's this doing?

Reply:
Based on your above comment, I'm guessing stack trace?